src/components/LiveSession/partials/LiveChannelCard.tsx

```diff
@@ -0,0 +1,67 @@
+import Image from 'next/image';
+import { FC } from 'react';
+import { FaWifi } from 'react-icons/fa';
+import { BsCheckCircleFill } from 'react-icons/bs';
+import formatNumber from '@/utils/formatNumber';
+
+interface LiveChannelCardProps {
+	image_url: string;
+	title: string;
+	liveTitle: string;
+	numberOfWatches: number;
+	profile: {
+		image: string;
+		name: string;
+	};
+}
+
+const LiveChannelCard: FC<LiveChannelCardProps> = (props) => {
+	const { image_url, title, liveTitle, numberOfWatches, profile } = props;
+
+	return (
+		<div className="bg-slate-900 h-fit w-fit flex flex-col gap-6 rounded-3xl cursor-pointer">
+			<div className="relative max-w-96 min-w-fit h-56 rounded-t-3xl">
+				<Image
```

Comment:
Always advised to add a height and width property on Image components

Reply:
it has fill to true for its width to change from max to min width ... this is because each card fill the grid width and the image follows the same width

Reply:
Actually, I was not aware of this. Thank you for highlighting that